code/glTF2Asset.h

```diff
@@ -670,41 +672,66 @@ namespace glTF2
         inline void SetData(uint8_t* data, size_t length, Asset& r);
     };
 
-    //! Holds a material property that can be a texture or a color
-    struct TexProperty
+    struct TextureInfo
     {
         Ref<Texture> texture;
-        vec4 color;
+        unsigned int index;
+        unsigned int texCoord = 0;
```

Comment:
I noticed other structs initialized values in a `SetDefault()` method. Is there anything invalid about setting the default value inline like this?

Reply:
Nope, not that I know of.